src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -124,11 +126,14 @@ class SceneryWindow final : public Window
             return std::find(std::begin(Entries), std::end(Entries), entry) != std::end(Entries);
         }
 
-        void AddEntry(const ScenerySelection& entry)
+        void AddEntry(const ScenerySelection& entry, const bool toBack)
```

Comment:
`const` arguments are pointless in C++, they are passed as copies anyway. Here it's an implementation detail, not important to the caller.